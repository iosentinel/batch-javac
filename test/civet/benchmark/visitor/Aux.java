package benchmark.visitor;

public class Aux {
	static int ctr = 0;

	static Tree buildTree(int i) {
		Tree t;
		if (i == 0)
			t = new Leaf(ctr++);
		else
			t = new Node(buildTree(i - 1), buildTree(i - 1));
		return t;
	}
}
