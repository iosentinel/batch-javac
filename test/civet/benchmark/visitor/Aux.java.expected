package benchmark.visitor;

public class Aux {
    
    public Aux() {
        super();
    }
    static int ctr = 0;
    
    static Tree buildTree(int i) {
        Tree t;
        if (i == 0) t = new Leaf(ctr++); else t = new Node(buildTree(i - 1), buildTree(i - 1));
        return t;
    }
}